### PR TITLE
Add users to database

### PR DESCRIPTION
**Pull request by aashishpiitk**
adding users to database

### CODE REVIEW

File: users/jobs/daily/scrape.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+
+
+            print("Processed {}".format(i + 12))
+            conn.commit()
+        conn.close()
+
+x = Job()
+x.execute(soup,c)
+#x.process_response_soup(soup,c)
```
Comment (vikrantmalik051) [inline users/jobs/daily/scrape.py]: Why is this comment added?

File: users/jobs/daily/scrape.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    'selnam1': '',
+    'mail': ''
+}
+
+payload1 = {
+    'typ': ['stud'] * 12,
+    'numtxt': '',
+    'sbm': ['Y'] * 12
+}
+TOTAL = 8385
```
Comment (vikrantmalik051) [inline users/jobs/daily/scrape.py]: I’m not sure if this is the total number of students now. It was the case a year ago. Have you checked whether this is the correct number?

**Comment by vikrantmalik051** [comment]
Also, next time, try to cherry pick the commits from which you write the code to retain the original author of the code.

**Comment by aashishpiitk** [comment]
> Also, next time, try to cherry pick the commits from which you write the code to retain the original author of the code.

I have done the required changes to total and comments
how to do this ,i.e. preserving the original author of commit

**Comment by vikrantmalik051** [comment]
> how to do this ,i.e. preserving the original author of commit

https://stackoverflow.com/questions/9339429/what-does-cherry-picking-a-commit-with-git-mean

**Comment by vikrantmalik051** [comment]
@aashishpiitk  Please squash the commits.

**Pull request merged**